Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/colcon_parallel_executor/executor/parallel.py", line 75, in execute
    loop.run_until_complete(future)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 636, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.10/asyncio/base_events.py", line 603, in run_forever
    self._run_once()
  File "/usr/lib/python3.10/asyncio/base_events.py", line 1871, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.10/selectors.py", line 469, in select
    fd_event_list = self._selector.poll(timeout, max_ev)
KeyboardInterrupt

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/colcon_core/executor/__init__.py", line 91, in __call__
    rc = await self.task(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/colcon_core/task/__init__.py", line 93, in __call__
    return await task_method(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/colcon_ros/task/ament_python/build.py", line 102, in build
    return await extension.build(additional_hooks=additional_hooks)
  File "/usr/lib/python3/dist-packages/colcon_core/task/python/build.py", line 85, in build
    available_commands = await self._get_available_commands(
  File "/usr/lib/python3/dist-packages/colcon_core/task/python/build.py", line 175, in _get_available_commands
    output = await check_output(
  File "/usr/lib/python3/dist-packages/colcon_core/subprocess.py", line 129, in check_output
    assert not rc, f'Expected {args} to pass: {stderr_data}'
AssertionError: Expected ['/usr/bin/python3', 'setup.py', '--help-commands'] to pass: Traceback (most recent call last):
  File "/home/burger/worren_hu_ws/src/navigate_to_goal/setup.py", line 5, in <module>
    setup(
  File "/home/burger/.local/lib/python3.10/site-packages/setuptools/__init__.py", line 153, in setup
    return distutils.core.setup(**attrs)
  File "/home/burger/.local/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 134, in setup
    ok = dist.parse_command_line()
  File "/home/burger/.local/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 481, in parse_command_line
    if self.handle_display_options(option_order):
  File "/home/burger/.local/lib/python3.10/site-packages/setuptools/dist.py", line 1120, in handle_display_options
    return _Distribution.handle_display_options(self, option_order)
  File "/home/burger/.local/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 684, in handle_display_options
    self.print_commands()
  File "/home/burger/.local/lib/python3.10/site-packages/setuptools/dist.py", line 896, in print_commands
    cmdclass = ep.resolve()
  File "/home/burger/.local/lib/python3.10/site-packages/pkg_resources/__init__.py", line 2456, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/burger/.local/lib/python3.10/site-packages/setuptools/command/develop.py", line 9, in <module>
    from setuptools.command.easy_install import easy_install
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load_unlocked
KeyboardInterrupt

